config.html

```diff
@@ -350,6 +379,84 @@
                             <span>seconds</span>
                         </td>
                     </tr>
+                    <tr>
+                        <td colspan="2">
+                            <label for="special_search">Activate the special search bar</label>
+                            <input data-originalvalue="" id="special_search" name="special_search" type="checkbox">
+
+                            <button id="toggleParams" onclick="toggleParams()">Show/Hide Parameters</button>
+                            <div id="params" style="display: none;">
```

Comment:
This configuration for special search looks a bit cluttered, maybe we can add some space between them?

Reply:
It's true that the parameters are very close to each other